docs/learn/guides/saas-clients/sfdc46/import-contacts-into-salesforce-using-ftp/README.md

```diff
@@ -0,0 +1,141 @@
+# Import Contacts Into Salesforce using FTP connector
+
+## About
+
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the 
+support of connectors. In this guide, we are mainly focusing on importing CSV file having contacts into Salesforce
+using FTP connector. 
+
+The `wso2/sfdc46` module allows you to perform CRUD operations for SObjects, query using SOQL, search using SOSL, and 
+describe SObjects and organizational data through the Salesforce REST API. Also it supports insert, upsert, update, 
+query and delete operations for CSV, JSON and XML data types which provides in Salesforce bulk API. It handles OAuth 
+2.0 authentication.
+
+The `wso2/ftp` module provides an FTP client and an FTP server listener implementation to facilitate an FTP connection 
+to a remote location.
+
+You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) Github organization.
+
+## What you'll build
+
+This application listens to a remote FTP location and when CSV file appears (This CSV file contains contacts should be 
+inserted to salesforce) it will fetch CSV file content and insert all the contacts in the CSV into Salesforce using
+`salesforceBulkClient` as a single batch. Then it will get the insertion result using `salesforceBulkClient` and the
+operation is successful, a success message is logged.
+
+![import contacts to sfdc using ftp](../../../../../assets/img/import_contacts_to_sfdc_using_ftp.png)
+
+## Prerequisites
+
+Link to download ballerina integrator.
+
+## Implementation
+
+The ballerina project should create for the integration use case explained above. Please follow the steps given below 
+to create the project and modules. You can learn about the ballerina project and modules in this [guide]().
```

Comment:
Are we pointing to https://ei.docs.wso2.com/en/latest/ballerina-integrator/develop/using-modules/